### PR TITLE
Add accessibility features

### PR DESCRIPTION
**Pull request by Wikitelmex**
- fixed page tittle
- fixed text alternatives
- fixed 3 h1 elements turned to h2 to fix text headings
- checked color contrast with edge tools, and external tools.
- added units "REM" instead "PX" to the font-size, to a proper scaling when user require it.
- fixed issues about scaling font size, in heights of cards and containers
- Interaction, order and visibility checked
- moving content, multimedia ckecked
- basic structure of the page checked

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -261,9 +261,9 @@ <h1 class="about">About me</h1>
         <footer>
             <div class="light">
                 <div class="footer-form">
-                    <h1 id="h1footer">
+                    <h2 id="h1footer">
                         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-                    </h1>
+                    </h2>
                     <form name="contact-form" class="portfolio-form" action="https://formspree.io/f/mnqlovey" method="POST">
```
Comment (SajjadAhmad14) [inline index.html:267]: Some buttons have a bad color contrast ratio. Please use [this](https://www.aditus.io/) tool to analyze it and fix issues.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Alex Castillo Cerros, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [x] GitHUb flow is used.
- [x] README is professional
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Wikitelmex) [reply index.html:267]: Nice tool, Thanks a lot, I fixed the contrast ratio

image

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Wikitelmex and @mike2611, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### [OPTIONAL] changes required

- [ ] Improve the description for your project in the README file (three lines may be a reference for you), try to explain what you did, what kind of technologies you used, what is the scope for your project, and so on.

## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**